frontend/annotation_resolver.py

```diff
@@ -161,19 +163,19 @@ def add_annotated_function_axioms(self, args_types, solver, annotated_function,
         args_annotations = annotated_function.args_annotations
         result_annotation = annotated_function.return_annotation
 
+        axioms = []
+
         if len(args_types) != len(args_annotations):
-            raise TypeError("The function expects {} arguments. {} were given.".format(len(args_annotations),
-                                                                                       len(args_types)))
+            return None
```

Comment:
No error is raised in this case, because we don't add this `None` to the solver. We just add no axioms for this built-in type.

so if we have
```python
x = [1,2,3]
x.append(1, 2)
```
which is invalid in terms of number of arguments, the `get_annotated_function_axioms` function will return None, and thus `call_axioms` will be empty, so doing an `Or` on an empty list will give `unsat`